src/core_modules/capture-core/components/WidgetRelatedStages/RelatedStagesActions/RelatedStagesActions.component.js

```diff
@@ -80,15 +84,27 @@ export const RelatedStagesActionsPlain = ({
         >
             <div className={classes.wrapper}>
                 {type === relatedStageStatus.LINKABLE ? Object.keys(mainOptionTranslatedTexts).map(key => (
-                    <Radio
-                        key={key}
-                        name={`related-stage-action-${key}`}
-                        checked={key === selectedAction}
-                        disabled={key === RelatedStagesActionTypes.LINK_EXISTING_RESPONSE && !linkableEvents.length}
-                        label={mainOptionTranslatedTexts[key]}
-                        onChange={(e: Object) => updateSelectedAction(e.value)}
-                        value={key}
-                    />
+                    <div className={classes.fieldWrapper} key={key}>
```

Comment:
It seems like this div changed the styling quite drastically. It would be nice if we could keep the design look the same.

Before:
<img width="1723" alt="image" src="https://github.com/dhis2/capture-app/assets/12842868/6d9f1990-8fd5-439c-9e9a-5921bc650d6d">

After:
<img width="1723" alt="image" src="https://github.com/dhis2/capture-app/assets/12842868/65330f0d-9c5e-4b31-9e5b-9b1b9144e6b0">